### PR TITLE
Opening the SF guide with commands #491

### PR DESCRIPTION
**Pull request by Redemption198**
From the suggestion of @oddare (https://github.com/TheBusyBiscuit/Slimefun4/issues/491), I've added the command to open the Slimefun Guide just typing "/sf open_guide". I also added the tab completion "open_guide" and a description of it.

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
You could add a permission as well.

**Comment by Redemption198** [comment]
If there is something that you don't like just let me know.

**Comment by oddare** [comment]
Dang, this is awesome! :D

**Comment by Redemption198** [comment]
I was looking at SlimefunCommand.java and I saw that the command that gives the books doesn't check for any permission, I was wondering why.

File: src/plugin.yml [hunk 1]
```diff
@@ -34,6 +34,9 @@ permissions:
   slimefun.command.versions:
     description: Allows you to do /sf versions
     default: op
+  slimefun.command.guide:
+    description: Allows you to obtain the Slimefun guide book
+    default: op
```
Comment (TheBusyBiscuit) [inline src/plugin.yml]: default: true

please

Comment (Redemption198) [reply src/plugin.yml]: Done. Shall I change open_guide default too?

Comment (TheBusyBiscuit) [reply src/plugin.yml]: No.

**Pull request merged**